gosheets.scss

```diff
@@ -1 +1,3 @@
 @import './base/variables';
+@import './base/grid';
```

Comment:
Should we alphabetize those imports?

Reply:
I like where your head is at, but we can't alphabetize imports. Styles will be imported in the order in which they are listed here. This gives us control over the way the styles cascade. The order here is crucial to how the styles will be compiled.

When sass compiles it doesn't really have access to what will be compiled after a specific point, only backwards. That means that if the variables were last, none of the files would have access to them.

Reply:
That makes totally sense. Thank you for the clarification! 👍🏻